src/core/src/core_logic/ConfigurePatchingProcessor.py

```diff
@@ -128,8 +128,6 @@ def __report_consolidated_configure_patch_status(self, status=Constants.STATUS_T
             error_msg = 'Error: ' + repr(error)
             self.composite_logger.log_error(error_msg)
             self.status_handler.add_error_to_status(error_msg, Constants.PatchOperationErrorCodes.DEFAULT_ERROR)
-            if Constants.ERROR_ADDED_TO_STATUS not in repr(error):
-                error.args = (error.args, "[{0}]".format(Constants.ERROR_ADDED_TO_STATUS))
```

Comment:
Why was this removed?

Reply:
Justififcation the point 2 in PR desc/first comment


Removed the statement to add Constants.ERROR_ADDED_TO_STATUS tag the error object in__report_consolidated_configure_patch_status() in ConfigurePatchingProcessor, because we only add the tag when there is a possibility of the same error getting added again to error objects from some other place. Since we do not raise or call any other function from within __report_consolidated_configure_patch_status(), this error will not be duplicated in error objects